src/Tooltip.jsx

```diff
@@ -1,44 +1,42 @@
 /**
  * @class Tooltip
- * @description A lightweight and responsive tooltip.
+ * @description The container of a lightweight and responsive tooltip.
  */
 import React from 'react';
 import PropTypes from 'prop-types';
 
-import Portal from 'react-minimalist-portal';
-import positions from './position';
+import TooltipBubble from './TooltipBubble';
 
-// default colors
-const defaultColor = '#fff';
-const defaultBg = '#333';
-
-const stopProp = e => e.stopPropagation();
-
-class Tooltip extends React.PureComponent {
+class Tooltip extends React.Component {
   static propTypes = {
+    // eslint-disable-next-line react/no-unused-prop-types
     direction: PropTypes.string,
-    className: PropTypes.string,
+    tooltipClassName: PropTypes.string,
```

Comment:
We already call the component Tooltip, so when creating a Tooltip component instance and assigning props, I feel they shouldn't need to be prefixed.